hermes_memory_provider/__init__.py

```
@@ -1602,6 +1602,37 @@ def _read_config_key(self, key: str) -> Any:
+            return False
+        config_path = os.path.join(hermes_home, "config.yaml")
+        if not os.path.exists(config_path):
+            return False
+        try:
+            import yaml
+        except ImportError:
+            return False
+        try:
+            with open(config_path) as f:
+                config = yaml.safe_load(f) or {}
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)